src/pages/character/[id].tsx

```diff
@@ -0,0 +1,110 @@
+import { useRouter } from 'next/router';
+import { api } from '../../utils/api';
+
+import {
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  Stack,
+  Heading,
+  Text,
+  Button,
+  Box,
+  Flex,
+  Center,
+  Image,
+  AspectRatio,
+} from '@chakra-ui/react';
+
+const Post = () => {
+  const router = useRouter();
+  const { id } = router.query;
+
+  // https://trpc.io/docs/nextjs#6-make-an-api-request
+  const characterRequest = api.characterRouter.getCharacter.useQuery({
+    id: id ? +id : 0,
+  });
+
+  const characterInfo = characterRequest.data;
+
+  return (
+    <Box w="100%" h="100vh" bg="black">
+      <Center pb={4}>
+        <Image src="/LordOfRingLogo.png" alt="Logo" width="300px" />
+      </Center>
+      <Center>
```

Comment:
Looking at the Figma, we want the card to be centered on the page---maybe try putting the image & the card in the same `Center` component & giving the image absolute positioning